<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFixtureDialog</class>
 <widget class="QDialog" name="CreateFixtureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>382</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="hL_name">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lE_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cB_icons"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cB_type"/>
     </item>
     <item>
      <widget class="QComboBox" name="cB_colors"/>
     </item>
     <item>
      <widget class="QLabel" name="l_start_channel">
       <property name="text">
        <string>Startchannel:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sB_start_cahnnel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hL_desc">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lE_description"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vL_scrollarea"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hL_buttons">
     <item>
      <widget class="QPushButton" name="pB_add_channel">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
